tripy/tripy/frontend/module/module.py

```diff
@@ -54,7 +54,8 @@ class Module:
     Base class used to define neural network modules.
     You can nest modules by assigning them as attributes of other modules.
 
-    Child modules or :class:`tripy.Parameter` s may be contained in Python ``list``\s or ``dict``\s.
+    Child modules, :class:`tripy.Parameter` s, or other callables/lambda functions may be contained
+    in Python ``list``\s or ``dict``\s.
     If using ``dict``\s, the keys must be strings.
```

Comment:
I believe the backslashes here were intended to escape a space, not the `s`. 
```suggestion
    in Python ``list``\ s or ``dict``\ s.
    
    If using ``dict``\ s, the keys must be strings.
```